tests/results/result_actions/OpenEditorActionTest.py

```diff
@@ -216,6 +217,22 @@ def test_open_files_at_position_no_position(self):
                 stdout=subprocess.PIPE
             )
 
+    def test_unknown_editor_warning(self):
+        logger = logging.getLogger()
+        uut = OpenEditorAction()
+        result_mock = Result.from_values(
+            'test', '', self.fa, line=None, column=None,
+        )
+        with unittest.mock.patch('subprocess.call'):
```

Comment:
To clear the wip after answering this question, commit witht he amend option and force push without making any changes. Alternatively, ping me on gitter.